src/Components/Overlay.js

```diff
@@ -11,9 +11,9 @@ const Overlay = () => {
   // We may have to many moving parts.  Do we need a currentTile if the player position is already tracking this?
```

Comment:
I think we may be able to get rid of this component completely to avoid excessive re-rendering.